product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -78,15 +78,16 @@ Once your identity provider is set up, you can view your connection status, ID,
 You need a verified domain so your users can have a streamlined login experience with their email address. 
 
 1. On the **Domains** tab, enter the domain name and select **Next: Verify Domain**. 
-1. Copy the TXT record and follow the instructions to add it as a TXT record on that domain within your DNS provider's management console.
-1. Select **Done**. 
+2. Copy the TXT record and follow the instructions in the on screen verify box, to add it as a TXT record on that domain within your DNS provider's management console.
```

Comment:
```suggestion
2. To add it as a TXT record on that domain, copy the TXT record and follow the instructions in the on-screen verify box in your DNS provider's management console.
```

Reply:
That is not correct. The on-screen verify box is in big animal. It has the instructions I have added above. For use on the third party site.